src/interfaces/swig/shogun.i

```diff
@@ -82,8 +82,8 @@
 %include "Boost_includes.i"
 
 %include "SGBase.i"
-%include "IO.i"
 %include "Library.i"
+%include "IO.i"
```

Comment:
@karlnapf @vigsterkr is it ok to swap the order? now that I depend on CList Library.i needs to come before so that swig knows of its existence.. It also means that Features.i should be above IO.i, but I am guessing that would cause issues?
Should probably rethink how these are split, since SGIO.h is used everywhere, but parsers aren't..